locked-asset/proxy_dex/src/sc_whitelist.rs

```diff
@@ -0,0 +1,28 @@
+elrond_wasm::imports!();
+
+#[elrond_wasm::module]
+pub trait ScWhitelistModule {
+    #[only_owner]
+    #[endpoint(addPairToIntermediate)]
+    fn add_pair_to_intermediate(&self, pair_address: ManagedAddress) {
+        let _ = self.intermediated_pairs().insert(pair_address);
+    }
+
+    #[only_owner]
+    #[endpoint(removeIntermediatedPair)]
+    fn remove_intermediated_pair(&self, pair_address: ManagedAddress) {
+        self.require_is_intermediated_pair(&pair_address);
+        let _ = self.intermediated_pairs().swap_remove(&pair_address);
+    }
+
+    fn require_is_intermediated_pair(&self, address: &ManagedAddress) {
```

Comment:
`require_is_intermediated_pair` -> `is_intermediated_pair` ?

Reply:
`is_intermediated_pair` would generally be a function that returns boolean, while the `require` prefix signals that an error will be thrown if the condition is not fulfilled.